lib/sessions.js

```diff
@@ -39,6 +39,12 @@ class ClientSession extends EventEmitter {
     }
 
     this.operationTime = null;
+
+    this.explicit = !!options.explicit;
+  }
+
+  isExplicit() {
```

Comment:
tbh I'm inclined to say we can just access the `.explicit` property explicitly, pun intended. Otherwise this is at very least a `get` accessor

Reply:
Ok, that is doable